purchase_quick/tests/test_quick_purchase.py

```diff
@@ -194,27 +195,6 @@ def test_several_po_for_one_product(self):
         with self.assertRaises(ValidationError):
             self.product_1.qty_to_process = 3.0
 
-    def test_purchaser_can_edit_products(self):
-        """
-        While in the quick purchase interface, a purchaser with no edit rights
-        on product.product can still edit product.product quick quantities
-        """
-        po = self.env["purchase.order"].create({"partner_id": self.partner.id})
-        ctx = {
-            "parent_id": po.id,
-            "parent_model": "purchase.order",
-            "quick_access_rights_purchase": 1,
-        }
-        product = self.env.ref("product.product_product_8")
-        with self.assertRaises(AccessError):
-            product.with_user(self.user).write(
-                {"qty_to_process": 5.0, "quick_uom_id": self.uom_unit.id}
-            )
-        product_in_quick_edit = product.with_user(self.user).with_context(**ctx)
-        product_in_quick_edit.write(
-            {"qty_to_process": 5.0, "quick_uom_id": self.uom_unit.id}
-        )
-
```

Comment:
why did you remove this test ?

Reply:
I don't remember exactly. (It was 9 monthes ago ;-) !), As far as I remember, there were messy incompatibilities with other module in the same repo, that change access rights.